ebib.el

```diff
@@ -4095,23 +4101,9 @@ special manner."
 		  ;; a cond.
                   (pfx
                    (ebib--edit-normal-field field init-contents))
```

Comment:
`ebib--edit-normal-field` must be adapted, because it returns a boolean, which is now stored in the field being edited.

In the code you're replacing, the edit helper functions themselves were responsible for storing the new value in the database. I do think having the helper functions return the new value and let `ebib-edit-field` store it) is cleaner, so I'm OK with the change, it's just that `ebib--edit-normal-field` should be modified. (And perhaps renamed, because I don't think the name still reflects its purpose.)

Reply:
Done in a ca62f6c.

My solution moves all the database stuff (setting the field value, or deleting it if the returned value is empty) up to the calling function `ebib-edit-field`. As far as I can tell, it works correctly, but you may want to check it. 

I'm not sure about the new name: `ebib--edit-literal-field`. It seemed sort of appropriate, but maybe you can think of something better?